### PR TITLE
Current db in session

### PR DESCRIPTION
**Pull request by bheni**

### CODE REVIEW

File: server/context.go [hunk 1]
```diff
@@ -75,20 +75,29 @@ func (s *SessionManager) NewSession(ctx context.Context, conn *mysql.Conn) error
...
+
 func (s *SessionManager) session(conn *mysql.Conn) sql.Session {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 	return s.sessions[conn.ConnectionID]
 }
 
 // NewContext creates a new context for the session at the given conn.
-func (s *SessionManager) NewContext(conn *mysql.Conn) (*sql.Context, error) {
+func (s *SessionManager) NewContext(conn *mysql.Conn, initialDB string) (*sql.Context, error) {
```
Comment (zachmu) [inline server/context.go:96]: New param not used?

File: sql/session.go [hunk 2]
```diff
@@ -51,15 +60,33 @@ type Session interface {
 	WarningCount() uint16
 }
 
+/*var defaultDB =""
```
Comment (zachmu) [inline sql/session.go:63]: Remove?

File: server/context.go (hunk 1 above)
Comment (bheni) [reply server/context.go:96]: good catch. fixed.

File: sql/session.go (hunk 2 above)
Comment (bheni) [reply sql/session.go:63]: fixed

**Pull request merged**